### DISCUSSION TITLE
Error in .order_seqlevels(chrom_sizes[, "chrom"]): !anyNA(m32) is not TRUE

### DISCUSSION THREAD

**Discussion by DanieleMuraro** [stuart-lab/signac#845] [Q&A] [answered]
Hi,

I have tried to run the tutorial "Joint RNA and ATAC analysis: 10x multiomic"

https://satijalab.org/signac/articles/pbmc_multiomic.html

When running 

seqlevelsStyle(annotation) <- "UCSC"

I am getting the following error (see attached):

Error in .order_seqlevels(chrom_sizes[, "chrom"]): !anyNA(m32) is not TRUE

Does anybody know what it is due to?

Thanks,

Daniele
[tutorial.pdf](https://github.com/timoast/signac/files/7399550/tutorial.pdf)

**Comment by timoast** [comment, marked as answer]
See https://github.com/timoast/signac/issues/826

**Reply by DanieleMuraro** [reply]
@timoast thank you so much for your help. Was this issue solved? 

As far as I am understanding, one to the issues was inverting the order of the following assignments: 

genome(annotation) <- "hg38"
seqlevelsStyle(annotation) <- "UCSC"

another was updating the version of GenomeInfoDb.

I am currently using 

packageVersion('GenomeInfoDb')
'1.31.0'

but when running 

pbmc <- TSSEnrichment(pbmc)

I now get

Extracting TSS positions

Extracting fragments at TSSs


Computing TSS enrichment score

Error in ecdf(x = object$TSS.enrichment): 'x' must have 1 or more non-missing values
Traceback:

1. TSSEnrichment(pbmc)
2. TSSFast(object = object, assay = assay, tss.positions = tss.positions, 
 .     process_n = process_n, verbose = verbose)
3. ecdf(x = object$TSS.enrichment)
4. stop("'x' must have 1 or more non-missing values")


Should I modify the pbmc object?

Thanks,

Daniele

[Uploading tutorial.pdf…]()

**Reply by timoast** [reply]
This should be resolved by updating GenomeInfoDb, if not please open an issue with your full code and output of `sessionInfo()`

**Reply by DanieleMuraro** [reply]
Hi,

I am using GenomeInfoDb_1.31.0 (see attached).
As far as I am understanding this is the latest version.
Thanks again :-)

[tutorial.pdf](https://github.com/timoast/signac/files/7442985/tutorial.pdf)

**Reply by timoast** [reply]
It looks like the chromosome naming in the gene annotation does not match the naming in your fragment file. Please ensure the naming is correct before adding the annotation to the assay and running `TSSEnrichment()`. If you have further problems, please open an issue
